site/docs/platforms.md

```diff
@@ -113,3 +113,117 @@ command-line flags:
 
 *  `--host_platform` - defaults to `@bazel_tools//platforms:host_platform`
 *  `--platforms` - defaults to `@bazel_tools//platforms:target_platform`
+
+## Skipping incompatible targets
+
+When building for a specific target platform it is often desirable to skip
+targets that will never work on that platform. For example, your Windows device
+driver is likely going to generate lots of compiler errors when building on a
+Linux machine with `//...`. Use the
+[`target_compatible_with`](be/common-definitions.html#common.target_compatible_with)
+attribute to tell bazel what target platform constraints your code has.
+
+The simplest use of this attribute restricts a target to a single platform.
+The target will not be built for any platform that doesn't satisfy all of the
+constraints. The following example restricts `win_driver_lib.cc` to 64-bit
+Windows.
+
+```python
+cc_library(
+    name = "win_driver_lib",
+    srcs = "win_driver_lib.cc",
+    target_compatible_with = [
+        "@platforms//cpu:x86_64",
+        "@platforms//os:windows",
+    ],
+)
+```
+
+When building for anything but 64-bit Windows we say that `:win_driver_lib` is
+incompatible.  Incompatibility is transitive. Any targets that transitively
+depend on an incompatible target are themselves considered incompatible.
+
+### When are targets skipped?
+
+Targets are skipped when they are considered incompatible and included in the
+build as part of a glob. For example, the following two invocations skip any
```

Comment:
A glob is something very different (https://docs.bazel.build/versions/master/be/functions.html#glob), this is a target pattern expansion (https://docs.bazel.build/versions/master/query.html#target-patterns).

Reply:
Done